### ISSUE TITLE
 Clear all the notifications from the notification itself:

### ISSUE THREAD

**Issue by kobion** [katzer/cordova-plugin-local-notifications#1748]
When clicking on an action button on a local notification, the event won't fire until I open the app.
I'm trying to clear all the notifications from the notification itself:
```
cordova.plugins.notification.local.on('yes', function() {
    cordova.plugins.notification.local.cancelAll(function (response) {
        console.log('Notifications cleared!');
    });   
});
```
How can it work outside the app?

## Your Environment

* Plugin version: 0.9.0-beta.3
* Platform: Android
* Device manufacturer / model: Android Samsung A7
* Cordova version (```cordova -v```): 8.1.1
* Cordova platform version (```cordova platform ls```): 7.1.4

## Steps to Reproduce

```
    cordova.plugins.notification.local.schedule({
        id: id,
        title: 'TEST',
        text: 'ITS WORKING!',
        trigger: { in: seconds, unit: 'second' },
        vibrate: true,
        foreground: false,
        actions: [
            { id: 'yes', title: 'test yes btn' },
        ]        
    }, function(response){
        console.log(response ? 'Scheduled test notification' : 'Test notification failed to schedule');
    });  
```

**Comment by CreativeArtDesign** [comment]
I got the same issue.

**Comment by Tawpie** [comment]
you don't show your code for handling the action buttons, this should have been added with cordova.plugins.notification.local.addActions to the action category before you schedule the notification. Without an action, the button doesn't have anything to do when clicked.

**Comment by CreativeArtDesign** [comment]
This code works but the sound does not work? I have the sms.mp3 in my assets folder and as i see it the file is on the right place.
What are I missing?I tried also to set the application on front but does not work, I mean when i have this code added to my schedule:
 foreground: true, <- Added but not working.

I expected that when i choice yes or no the application coming up from background.Or do i need other plugin for that.. I am using ionic 4

import { Component } from '@angular/core';import { LocalNotifications } from '@ionic-native/local-notifications/ngx';import { Storage } from '@ionic/storage';import { NavController, AlertController, Platform } from '@ionic/angular';import { ToastController } from '@ionic/angular';@Component({ selector: 'app-home', templateUrl: 'home.page.html', styleUrls: ['home.page.scss'],})export class HomePage { constructor( public navController: NavController, public navCtrl: NavController, private plt: Platform, private localNotifications: LocalNotifications, private alertController: AlertController, private toastController: ToastController) { this.localNotifications.on('trigger').subscribe(notification => { this.presentToast(); }); this.localNotifications.on('yes').subscribe(notification => { this.presentAlert(notification.data.meetingId); }); this.localNotifications.on('no').subscribe(notification => { this.presentAlert(notification.data.meetingId); }); }// TRIGGER async presentToast() { const toast = await this.toastController.create({ message: 'Stop the Sound!', duration: 2000 }); toast.present(); } async presentAlert(test) { const alert = await this.alertController.create({ header: 'Alert', subHeader: 'Subtitle', message: test, buttons: ['OK'] }); await alert.present(); }// On load or every check! scheduleNotification() { this.localNotifications.schedule({ id: 1, led: { color: '#FF00FF', on: 500, off: 500 }, title: 'Spela upp Athan - 1', text: 'Abdelbaset Athan spela upp.', trigger: {at: new Date(new Date().getTime() + 3000)}, data: { meetingId:"Athan - 1" }, sound: 'file://assets/sms.mp3', actions: [ { id: 'yes', title: 'Ok, fortsätt lyssna!' }, { id: 'no', title: 'Stopa Athan' } ] }); }}
    Den måndag 11 februari 2019 18:59:45 EET, Tawpie <notifications@github.com> skrev:  
 
 
you don't show your code for handling the action buttons, this should have been added with cordova.plugins.notification.local.addActions to the action category before you schedule the notification. Without an action, the button doesn't have anything to do when clicked.
